### PR TITLE
Updates for latest Lucky

### LINKED ISSUE #536: Update generated apps to use `mount` again
Now that this is merged in https://github.com/luckyframework/lucky/pull/1226

Any spot that mentions `m` should just be changed to `mount`.

### LINKED ISSUE #537: Update generated apps with new Avram Credentials
https://github.com/luckyframework/avram/pull/433

### PR DESCRIPTION
**Pull request by jwoertink**
Fixes #536 
Fixes #537 

These are some updates needed for the next version of Lucky. It includes
* Updating the Avram config to use Credentials
* Renaming `m` -> `mount`

### CODE REVIEW

**Review by paulcsmith — COMMENTED** [review]
Looks good! There is an odd line https://github.com/luckyframework/lucky_cli/pull/547/checks?check_run_id=1064226166#step:13:235 where it says shard "crystar". Which I don't think is a shard. Maybe that's what is causing the spec failure?

**Comment by jwoertink** [comment]
That one is https://github.com/naqvis/crystar but I'm not sure what's requiring that. It's a dependency of something that the default project installs. 🤷‍♂️

File: shard.override.yml [hunk 1]
```diff
@@ -3,9 +3,12 @@
...
-#   lucky:
-#     github: luckyframework/lucky
-#     branch: master
+dependencies:
+  lucky:
+    github: luckyframework/lucky
+    branch: master
+  wordsmith:
+    github: luckyframework/wordsmith
+    version: ~> 0.2.1
```
Comment (paulcsmith) [inline shard.override.yml:14]: Could you add a newline here?

**Review by paulcsmith — APPROVED** [review]
LGTM!

**Pull request merged**